src/Youtube.js

```diff
@@ -291,10 +300,6 @@ THE SOFTWARE. */
     src: function(src) {
       if (src) {
         this.setSrc({ src: src });
-
-        if (this.options_.autoplay && !_isOnMobile) {
```

Comment:
This is already handled in `this.setSrc`, `this.play()` was already being called twice in the tech.